modules/MicroMegaWattBSP/MicroMegaWattBSP.cpp

```diff
@@ -11,31 +11,31 @@ void MicroMegaWattBSP::init() {
         return;
     }
 
-    invoke_init(*p_powermeter);
```

Comment:
Since the powermeter impl init() is empty, I would not delete this line

Reply:
added